core/msd/src/main/rdl/Workload.rdli

```diff
@@ -145,4 +145,36 @@ resource BulkWorkloadResponse POST "/workloads" (name=getWorkloadsByDomainAndSer
         ResourceError UNAUTHORIZED;
         ResourceError TOO_MANY_REQUESTS;
     }
-}
\ No newline at end of file
+}
+
+// Api to discover an additional instance which can have static or dynamic or both IPs
+resource Workloads PUT "/domain/{domainName}/service/{serviceName}/workload/discover/instance" (name=postCompositeInstance) {
+    DomainName domainName;      // name of the domain
+    EntityName serviceName;     // name of the service
+    CompositeInstance instance; // Generic instance
+    authorize ("update", "{domainName}:service.{serviceName}");
+    expected NO_CONTENT;
+    exceptions {
+        ResourceError BAD_REQUEST;
+        ResourceError NOT_FOUND;
+        ResourceError FORBIDDEN;
+        ResourceError UNAUTHORIZED;
+        ResourceError TOO_MANY_REQUESTS;
+    }
+}
+
+// Api to delete an additional instance which can have static or dynamic or both IPs
+resource Workloads DELETE "/domain/{domainName}/service/{serviceName}/workload/discover/instance" (name=deleteCompositeInstance) {
+    DomainName domainName;      // name of the domain
+    EntityName serviceName;     // name of the service
+    CompositeInstance instance; // Generic instance
```

Comment:
I'm not sure if we really want a DELETE api with a body. Even if the server supports it, there is no guarantee if the request going through some other proxy service before hitting the server will maintain the body or just simply drop it. So my recommendation would be to redesign the API to not have any body.